tests/unit/clients/cassandra_tests.py

```diff
@@ -67,6 +77,13 @@ def test_credentials(self):
 
 class CassandraSessionAdapterTests(unittest.TestCase):
     def setUp(self):
+        # cleaning up prom registry
+        collectors = list(REGISTRY._collector_to_names.keys())
```

Comment:
:no_bell: This is an internal method which makes me worried that this test will be fragile. Is there any way we can do this without using the `_collector_to_names` attribute?

Reply:
I think I'm missing why we can't just import the metrics from the Cassandra client and clear them directly. We're not writing tests that check the values of any other metrics other than the ones in use by Cassandra, I'm not sure it makes sense to reset them all (if you can)